.github/actions/integration/candy-machine-core/action.yml

```diff
@@ -0,0 +1,27 @@
+# Implicit dependencies
+# - actions/checkout@v2
+# - ./.github/actions/install-linux-build-deps
+# - ./.github/actions/install-solana
+# - ./.github/actions/install-rust
+
+name: Candy Machine Core Integration Dependencies
+description: Builds the dependencies for the Candy Machine Core integration test
+runs:
+  using: 'composite'
+  steps:
+    # Build Rust Program
+    - uses: ./.github/actions/build-program
+      with:
+        name: candy-machine-core
+    - uses: ./.github/actions/build-program
+      with:
+        name: token-metadata
+
+    # Install JS SDK deps
+    - uses: ./.github/actions/yarn-install-and-build
+      with:
+        cache_id: sdk-candy-machine-core
+        working_dir: ./candy-machine-core/js
+        build_token_metadata: true
+        build_auction_house: true
```

Comment:
Why does this require building Auction House?

Reply:
I think I just duplicated how we had the behavior already? I think it might be a consequence of how we had workspaces on in the past and it may not be needed anymore?